packages/core/test/__snapshots__/IconField.js.snap

```diff
@@ -28,15 +29,15 @@ exports[`IconField renders 1`] = `
   color: inherit;
   background-color: transparent;
   border-radius: 2px;
-  border-width: 0px;
+  border-width: 1px;
   border-style: solid;
+  border-color: #c0cad5;
```

Comment:
this snap file also has `border-color` twice a few times

Reply:
Addressed in https://github.com/priceline/design-system/pull/683